src/h_matrix.cpp

```diff
@@ -994,7 +994,7 @@ template<typename T> HMatrix<T> * HMatrix<T>::subset(
         tmpMatrix->ownClusterTrees(true, true);
         if(this->isRkMatrix()) {
           tmpMatrix->rk(const_cast<RkMatrix<T>*>(rk()->subset(tmpMatrix->rows(), tmpMatrix->cols())));
```

Comment:
OK

Reply:
Non on aurait pas du arriver jusque la. Il manque `if(m->isNull())` en amont. Si j'avais pas peur de tout peter j'ajouterais un `assert(!this->isNull())` au debut de `subset`.